### ISSUE TITLE
bug: testnet-10 pigeon v0.7.0 account sequence mismatch error

### ISSUE THREAD

**Issue by verabehr** [palomachain/paloma#456]
# What is happening?
<details>
  <summary>Section description</summary>
<i>Provide as much context as you can. Provide relevant software versions, screenshots, copy & paste error messages and so on.
Give as much context as you can to make it easier for the developers to figure what is happening.</i>
</details>

pigeon errors when broadcasting signatures. Pigeon doesn't stop and validator does not get jailed. I restarted my pigeon after first seeing this error, but am seeing this consistently for the past ~24 hours

```
Aug 30 13:32:47 localhost pigeon[124000]: time="2022-08-30T13:32:47Z" level=error msg="couldn't broadcast signatures and process attestation" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 449, got 448: incorrect account sequence: invalid request" message-ids="[189]" queue-name=EVM/eth-main/evm-turnstone-message signed-messages="[map[id:189 signature:[96 61 39 184 59 3 176 111 184 152 91 166 95 180 77 104 145 193 115 55 25 71 255 15 55 74 251 95 65 204 29 230 120 164 143 211 245 163 10 32 154 61 65 128 22 135 37 20 162 224 70 158 161 254 48 251 205 34 79 156 146 253 158 247 1]]]"
Aug 30 13:32:47 localhost pigeon[124000]: time="2022-08-30T13:32:47Z" level=error msg="error returned in process loop" err="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 449, got 448: incorrect account sequence: invalid request"
Aug 30 13:33:40 localhost pigeon[124000]: time="2022-08-30T13:33:40Z" level=info msg="relayer loop"
```

# How to reproduce?
<details>
  <summary>Section description</summary>
  <i>Please write detailed steps of what you were doing for this bug to appear.</i>
</details>

run pigeon and paloma

# What is the expected behaviour?
<details>
  <summary>Section description</summary>
<i>If you know, please write down what is the expected behaviour. If you don't know, that's ok. We can have a discussion in comments.</i>
</details>

pigeon can broadcast signatures

**Comment by kj89** [comment]
I can confirm this bug.My validator is constantly getting jailed after having of sequence errors
```
-- Logs begin at Tue 2022-09-06 14:36:17 CEST, end at Wed 2022-09-07 19:15:47 CEST. --
Sep 07 18:55:55 big-mamba pigeon[2147929]: time="2022-09-07T18:55:55+02:00" level=info msg="relayer loop"
Sep 07 18:56:55 big-mamba pigeon[2147929]: time="2022-09-07T18:56:55+02:00" level=info msg="relayer loop"
Sep 07 18:57:55 big-mamba pigeon[2147929]: time="2022-09-07T18:57:55+02:00" level=info msg="relayer loop"
Sep 07 18:58:55 big-mamba pigeon[2147929]: time="2022-09-07T18:58:55+02:00" level=info msg="relayer loop"
Sep 07 18:59:55 big-mamba pigeon[2147929]: time="2022-09-07T18:59:55+02:00" level=info msg="relayer loop"
Sep 07 19:00:55 big-mamba pigeon[2147929]: time="2022-09-07T19:00:55+02:00" level=info msg="relayer loop"
Sep 07 19:01:55 big-mamba pigeon[2147929]: time="2022-09-07T19:01:55+02:00" level=info msg="relayer loop"
Sep 07 19:01:56 big-mamba pigeon[2147929]: time="2022-09-07T19:01:56+02:00" level=info msg="providing evidence for messages" messages-to-provide-evidence-for="[1105]" queue-name=EVM/eth-main/validators-balances
Sep 07 19:01:58 big-mamba pigeon[2147929]: time="2022-09-07T19:01:58+02:00" level=info msg="got balance" balance=104340254502457325 evm-address=0x63798AD4eb791a8247Bb522bCE38062E41F7CE26 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=108706100975042226 evm-address=0x1Dd71EE9e7b1CfF10CCb7E12520999c2d4fB45e5 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=92447573037736591 evm-address=0xf95837F321d1647dAB446eBB90ea578B39d02efe height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=100100000000000000 evm-address=0xa49f65cf05DcCe7c92621Ed41CBa9E4E09a5D618 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=130075920755128705 evm-address=0xE12f475763598Aa093EA0A8F0F9cB7CAA11502bC height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=140370628949839776 evm-address=0x12e3AC8ac7d8754590EaCb0413C91339815CD7b5 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=99788990701308710 evm-address=0xd157ADfF6beA9445f4bDB476eEcF44021F99a9Eb height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:01:59 big-mamba pigeon[2147929]: time="2022-09-07T19:01:59+02:00" level=info msg="got balance" balance=111030433987739757 evm-address=0xB08319a5e41f4D56F7D15724776143BE494aF16B height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=110442306824798309 evm-address=0x11b6c349D67244e2d62D1CB2f8460226b492CcEb height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=132883697582177251 evm-address=0x63f55bc560E981d53E1f5bb3643e3a96D26fc635 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=114674696470365417 evm-address=0xfC0f1E32e0Db02Cd81dE8F662A51E2C285d5228D height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=126319876379961938 evm-address=0x7e24006Be85DbEf1d8BFBa862Cc9aF9F269D35aF height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=83355630049182612 evm-address=0xB861a44EaeF847652c1d5f3b8aFE8661BcE58B16 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=118437242624856506 evm-address=0x279fC7c0ebEf3328d3899FE73464347A764C5f2C height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:00 big-mamba pigeon[2147929]: time="2022-09-07T19:02:00+02:00" level=info msg="got balance" balance=132620215308067817 evm-address=0x0Ba5F0e6BDbc820EC58BC44f9c96A8A515Fb8Cfc height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:01 big-mamba pigeon[2147929]: time="2022-09-07T19:02:01+02:00" level=info msg="got balance" balance=145228027279022044 evm-address=0xeE83770168D4EE756e74562C17BDED88091519D0 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:01 big-mamba pigeon[2147929]: time="2022-09-07T19:02:01+02:00" level=info msg="got balance" balance=145372216831631965 evm-address=0x7b7287710104F4E7F5fF4F6BD2dDD399cAb2D099 height=15491483 nearest-to-time="2022-09-07 17:00:53.087521244 +0000 UTC" queue-type-name=EVM/eth-main/validators-balances
Sep 07 19:02:55 big-mamba pigeon[2147929]: time="2022-09-07T19:02:55+02:00" level=info msg="relayer loop"
Sep 07 19:03:55 big-mamba pigeon[2147929]: time="2022-09-07T19:03:55+02:00" level=info msg="relayer loop"
Sep 07 19:04:55 big-mamba pigeon[2147929]: time="2022-09-07T19:04:55+02:00" level=info msg="relayer loop"
Sep 07 19:05:55 big-mamba pigeon[2147929]: time="2022-09-07T19:05:55+02:00" level=info msg="relayer loop"
Sep 07 19:06:31 big-mamba pigeon[2147929]: time="2022-09-07T19:06:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 10338, got 10337: incorrect account sequence: invalid request"
Sep 07 19:06:55 big-mamba pigeon[2147929]: time="2022-09-07T19:06:55+02:00" level=info msg="relayer loop"
Sep 07 19:07:01 big-mamba pigeon[2147929]: time="2022-09-07T19:07:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 10339, got 10338: incorrect account sequence: invalid request"
Sep 07 19:07:31 big-mamba pigeon[2147929]: time="2022-09-07T19:07:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:07:55 big-mamba pigeon[2147929]: time="2022-09-07T19:07:55+02:00" level=info msg="relayer loop"
Sep 07 19:08:01 big-mamba pigeon[2147929]: time="2022-09-07T19:08:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:08:31 big-mamba pigeon[2147929]: time="2022-09-07T19:08:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 10340, got 10339: incorrect account sequence: invalid request"
Sep 07 19:08:55 big-mamba pigeon[2147929]: time="2022-09-07T19:08:55+02:00" level=info msg="relayer loop"
Sep 07 19:09:01 big-mamba pigeon[2147929]: time="2022-09-07T19:09:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 10341, got 10340: incorrect account sequence: invalid request"
Sep 07 19:09:31 big-mamba pigeon[2147929]: time="2022-09-07T19:09:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:09:55 big-mamba pigeon[2147929]: time="2022-09-07T19:09:55+02:00" level=info msg="relayer loop"
Sep 07 19:10:01 big-mamba pigeon[2147929]: time="2022-09-07T19:10:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:10:31 big-mamba pigeon[2147929]: time="2022-09-07T19:10:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 10342, got 10341: incorrect account sequence: invalid request"
Sep 07 19:10:55 big-mamba pigeon[2147929]: time="2022-09-07T19:10:55+02:00" level=info msg="relayer loop"
Sep 07 19:10:56 big-mamba pigeon[2147929]: time="2022-09-07T19:10:56+02:00" level=info msg="messages to sign" message-ids="[1106]" queue-name=EVM/eth-main/evm-turnstone-message
Sep 07 19:10:56 big-mamba pigeon[2147929]: time="2022-09-07T19:10:56+02:00" level=info msg="signed a message" message-id=1106
Sep 07 19:10:56 big-mamba pigeon[2147929]: time="2022-09-07T19:10:56+02:00" level=info msg="signed messages" message-ids="[1106]" queue-name=EVM/eth-main/evm-turnstone-message signed-messages="[map[id:1106 signature:[215 47 155 126 38 142 1 104 136 175 170 184 236 37 95 37 204 94 245 168 215 146 149 208 40 231 114 42 62 150 7 180 87 22 168 196 215 163 61 134 238 160 153 109 48 21 74 211 37 2 196 119 211 161 9 231 101 15 159 242 148 89 244 168 0]]]"
Sep 07 19:11:01 big-mamba pigeon[2147929]: time="2022-09-07T19:11:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:11:02 big-mamba pigeon[2147929]: time="2022-09-07T19:11:02+02:00" level=error msg="couldn't broadcast signatures and process attestation" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request" message-ids="[1106]" queue-name=EVM/eth-main/evm-turnstone-message signed-messages="[map[id:1106 signature:[215 47 155 126 38 142 1 104 136 175 170 184 236 37 95 37 204 94 245 168 215 146 149 208 40 231 114 42 62 150 7 180 87 22 168 196 215 163 61 134 238 160 153 109 48 21 74 211 37 2 196 119 211 161 9 231 101 15 159 242 148 89 244 168 0]]]"
Sep 07 19:11:02 big-mamba pigeon[2147929]: time="2022-09-07T19:11:02+02:00" level=error msg="error returned in process loop" err="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:11:31 big-mamba pigeon[2147929]: time="2022-09-07T19:11:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:11:55 big-mamba pigeon[2147929]: time="2022-09-07T19:11:55+02:00" level=info msg="relayer loop"
Sep 07 19:11:56 big-mamba pigeon[2147929]: time="2022-09-07T19:11:56+02:00" level=info msg="messages to sign" message-ids="[1106]" queue-name=EVM/eth-main/evm-turnstone-message
Sep 07 19:11:56 big-mamba pigeon[2147929]: time="2022-09-07T19:11:56+02:00" level=info msg="signed a message" message-id=1106
Sep 07 19:11:56 big-mamba pigeon[2147929]: time="2022-09-07T19:11:56+02:00" level=info msg="signed messages" message-ids="[1106]" queue-name=EVM/eth-main/evm-turnstone-message signed-messages="[map[id:1106 signature:[215 47 155 126 38 142 1 104 136 175 170 184 236 37 95 37 204 94 245 168 215 146 149 208 40 231 114 42 62 150 7 180 87 22 168 196 215 163 61 134 238 160 153 109 48 21 74 211 37 2 196 119 211 161 9 231 101 15 159 242 148 89 244 168 0]]]"
Sep 07 19:12:01 big-mamba pigeon[2147929]: time="2022-09-07T19:12:01+02:00" level=error msg="couldn't broadcast signatures and process attestation" error="context canceled" message-ids="[1106]" queue-name=EVM/eth-main/evm-turnstone-message signed-messages="[map[id:1106 signature:[215 47 155 126 38 142 1 104 136 175 170 184 236 37 95 37 204 94 245 168 215 146 149 208 40 231 114 42 62 150 7 180 87 22 168 196 215 163 61 134 238 160 153 109 48 21 74 211 37 2 196 119 211 161 9 231 101 15 159 242 148 89 244 168 0]]]"
Sep 07 19:12:01 big-mamba pigeon[2147929]: time="2022-09-07T19:12:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:12:31 big-mamba pigeon[2147929]: time="2022-09-07T19:12:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:12:55 big-mamba pigeon[2147929]: time="2022-09-07T19:12:55+02:00" level=info msg="relayer loop"
Sep 07 19:13:01 big-mamba pigeon[2147929]: time="2022-09-07T19:13:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:13:31 big-mamba pigeon[2147929]: time="2022-09-07T19:13:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:13:55 big-mamba pigeon[2147929]: time="2022-09-07T19:13:55+02:00" level=info msg="relayer loop"
Sep 07 19:14:01 big-mamba pigeon[2147929]: time="2022-09-07T19:14:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:14:31 big-mamba pigeon[2147929]: time="2022-09-07T19:14:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = account sequence mismatch, expected 10344, got 10343: incorrect account sequence: invalid request"
Sep 07 19:14:55 big-mamba pigeon[2147929]: time="2022-09-07T19:14:55+02:00" level=info msg="relayer loop"
Sep 07 19:15:01 big-mamba pigeon[2147929]: time="2022-09-07T19:15:01+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
Sep 07 19:15:31 big-mamba pigeon[2147929]: time="2022-09-07T19:15:31+02:00" level=error msg="error while trying to keep pigeon alive" error="rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: validator palomavaloper13uslh0y22ffnndyr3x30wqd8a6peqh25m8p743 cannot be a pigeon: validator is jailed: invalid request"
```

**Comment by kj89** [comment]
I can confirm since I have stopped re-stake script, issue is not showing anymore

**Comment by kj89** [comment]
We will try setting `broadcast-mode block` to re-stake as one of community members suggested in the telegram chat. and will update on our obseravations

![image](https://user-images.githubusercontent.com/50621007/189091311-e43f2cd2-4858-4667-b027-1f5d1d5cadd2.png)

**Comment by Vizualni** [comment]
This usually happens when sending multiple TXs from the same key in parallel or when you send them in sequence but the TX hasn't been included in the blockchain.

**Comment by taariq** [comment]
Proposed fix in #469. Closing as now this is a feature.
